pages/common/docker-compose.md

```diff
@@ -0,0 +1,27 @@
+# docker-compose
+
+> Run and manage multi container docker applications
+
+- Create and start containers in the background using a local `docker-compose.yml` file:
+
+`docker-compose up -d`
+
+- Start containers, rebuild if necessary:
+
+`docker-compose up --build`
+
+- Start containers using a custom configuration file:
+
+`docker-compose --file {{path/to/config}} up`
+
+- Stop running containers:
```

Comment:
Does this stop all containers, or just some of them?

Reply:
Iv'e changed the description a bit to reflect these operations effect all containers

Reply:
Thanks :smiley_cat: